docs/workflows.rst

```diff
@@ -64,3 +67,59 @@ All the commands should be run on Pulp 3 machine.
             }
         ]
     }
+
+
+Reset migrated Pulp 3 data
+--------------------------
+
+.. note::
+    Nothing is getting reset or modified in any way in Pulp 2.
```

Comment:
s/is getting/will be/

Reply:
But we should probably mention that any Pulp 3 repositories / remotes / everything that were created without being migrated will be deleted as well, for the specified plugins.